src/Routers/UsersRouter.js

```diff
@@ -57,6 +57,17 @@ export class UsersRouter extends ClassesRouter {
           const user = response.results[0].user;
           // Send token back on the login, because SDKs expect that.
           user.sessionToken = sessionToken;
+
+          // Remove hidden properties.
+          for (var key in user) {
```

Comment:
That's the point, we won't. But those fields would only be stripped here, not everywhere we need them to be stripped